src/core_plugins/kbn_vislib_vis_types/public/controls/gauge_options.html

```diff
@@ -308,9 +379,13 @@
             </div>
           </div>
           <div class="kuiSideBarFormRow">
-            <label class="kuiSideBarFormRow__label" for="showScale">
-              Font Size (<span ng-bind="editorState.params.gauge.style.fontSize"></span>pt)
-            </label>
+            <label
+              class="kuiSideBarFormRow__label"
+              for="showScale"
+              i18n-id="kbnVislibVisTypes.controls.gaugeOptions.fontSizeLabel"
+              i18n-default-message="Font Size ({fontSizeSpan}pt)"
+              i18n-values="{ fontSizeSpan: '<span ng-bind=\'editorState.params.gauge.style.fontSize\'></span>' }"
```

Comment:
This looks wrong. This should rather just inject `editorState.params.gauge.style.fontSize` as a parameter and not that weird `span` ng-bind construct.

Also please rename the parameter name to `fontSize`, we should not keep element names (especially if they are not semantic elements at all) in the parameter name.